### PR TITLE
OADP-3183: Doc improvements as CherryPick for 4.13

### PR DESCRIPTION
**Pull request by anarnold97**
### Jira 

* [OADP-3183](https://issues.redhat.com/browse/OADP-3183)

This PR is a [cherry pick for 4.13](https://github.com/openshift/openshift-docs/pull/69508)

#### For OCP 4.13

* OCP 4.13 → branch/enterprise-4.13

### QE review

* [X] - please see [QE review of original PR](https://github.com/openshift/openshift-docs/pull/69508#issue-2045212009)

### CODE REVIEW

File: modules/oadp-installing-dpa-1-2-and-earlier.adoc [hunk 1]
```diff
@@ -394,13 +394,18 @@ oc get dpa dpa-sample -n openshift-adp -o jsonpath='{.status}'
...
-. Verify the `BackupStorageLocation` by running the following command:
+. Verify the backup storage location and confirm that the `PHASE` is `Available` by running the following command:
 +
 [source,terminal]
 ----
-oc get backupStorageLocation -n openshift-adp
+$ oc get backupStorageLocation -n openshift-adp
+----
+.Example output
+[source,yaml]
```
Comment (xenolinux) [inline modules/oadp-installing-dpa-1-2-and-earlier.adoc:404]: It says [source,yaml] but the output is not in yaml format. Should [source,yaml] be [source,terminal]?

File: modules/oadp-installing-dpa-1-2-and-earlier.adoc [hunk 2]
```diff
@@ -394,13 +394,18 @@ oc get dpa dpa-sample -n openshift-adp -o jsonpath='{.status}'
...
++
+----
 NAME           PHASE       LAST VALIDATED   AGE     DEFAULT
 dpa-sample-1   Available   1s               3d16h   true
 ----
 
-. Verify that the `PHASE` is in `Available`.
\ No newline at end of file
+. Verify that the `PHASE` is in `Available`.
```
Comment (xenolinux) [inline modules/oadp-installing-dpa-1-2-and-earlier.adoc:411]:
```suggestion
. Verify that the `PHASE` is `Available`.
```

File: modules/oadp-installing-dpa-1-3.adoc [hunk 3]
```diff
@@ -410,13 +410,18 @@ oc get dpa dpa-sample -n openshift-adp -o jsonpath='{.status}'
...
+[source,yaml]
++
+----
 NAME           PHASE       LAST VALIDATED   AGE     DEFAULT
 dpa-sample-1   Available   1s               3d16h   true
 ----
 
-. Verify that the `PHASE` is in `Available`.
\ No newline at end of file
+. Verify that the `PHASE` is in `Available`.
```
Comment (xenolinux) [inline modules/oadp-installing-dpa-1-3.adoc:427]:
```suggestion
. Verify that the `PHASE` is `Available`.
```

**Review by xenolinux — COMMENTED** [review]
Few suggestions to consider otherwise LGTM

Can you please provide the preview link?

/remove-label peer-review-needed
/remove-label peer-review-in-progress
/label peer-review-done

**Comment by anarnold97** [comment]
Thanks @xenolinux - good spots as always

**Comment by bscott-rh** [comment]
Merge review LGTM.

**Pull request merged**